package.json

```diff
@@ -30,5 +31,8 @@
     "@storybook/addons": "^4.1.11",
     "@babel/core": "^7.2.2",
     "babel-loader": "^8.0.5"
+  },
+  "config": {
+    "script-shell": "/bin/bash"
```

Comment:
Do you have windows subsystem installed since you have `/bin/bash` installed? :)

Reply:
removed, mkdirp already solved the problem